packages/local-cli/link/__tests__/link-test.js

```diff
@@ -226,9 +226,76 @@ describe('link', () => {
 
     const link = require('../link').func;
 
-    link(['react-native-blur'], context).then(() => {
+    link(['react-native-blur'], context, {}).then(() => {
       expect(registerNativeModule.mock.calls).toHaveLength(1);
       done();
     });
   });
+
+  it('should link only for specific platforms if --platforms is used', async () => {
+    jest.setMock('../getProjectDependencies', () => ['react-native-maps']);
+    jest.setMock('../../core/getPackageConfiguration', () => ({
```

Comment:
can you move `jest.mock` or `jest.doMock` instead?

Reply:
That works everywhere except here:

```js
jest.setMock('../../core/getPlatforms', () => ({
  android: { linkConfig: genericAndroidLinkConfig },
  ios: { linkConfig: genericIOSLinkConfig },
}));
```

Changing to `doMock` makes the test fail, do you know why? By the way, all the other tests from this file use also `setMock` so not sure that might be affecting the others somehow...

Reply:
I'd say get rid of as many `setMock` as you can. It's a legacy API and shouldn't be used (I'm not even sure what it does actually, this project is first time I encounter it in the wild)

Reply:
Done 💪